src/portal/templates/portal/index.html

```diff
@@ -0,0 +1,7 @@
+{% extends "portal/base.html" %}
+
+{% block content %}
+    <a>Hello, {{ user.get_full_name }}!</a>
+    <br>
+    <a href="{% url "logout" %}">Logout</a>
```

Comment:
"{% url "logout" %}" >> "{% url 'logout' %}"

^^ Just curious if it suppose to be like that